test/winter_onboarding_2021/fleet_management/healthcheck_test.clj

```diff
@@ -10,6 +10,6 @@
   (testing "Healthcheck for DB"
     (is (= {:status 200
             :headers {}
-            :body {:message "Everything OK!"
-                   :data [{:check 1}]}}
+            :body (str {:message "Everything OK!"
+                        :data [{:check 1}]})}
```

Comment:
Is there any need for stringifying the `:body`?
We are using `wrap-json-response` on the `/healthcheck` route `routes.clj`, right?